### DISCUSSION TITLE
Increase max context to 4k+ or a selectable amount?

### DISCUSSION THREAD

**Discussion by Doublespaced1** [LostRuins/koboldcpp#237] [Ideas]
I'm messing around with models that have 4k+ context windows such as Wes Brown's Cassandra models, and I noticed kobold still only seems to support 2048. Any way to bump that to 4096 or give us a selectable user amount?

**Comment by LostRuins** [comment]
Yes it can be done, You need to do 2 things:
1. Launch with --contextsize, e.g. `--contextsize 4096`, this will allocate more memory for a bigger context
2. Manually override the slider values in kobold Lite, this can be easily done by just clicking the textbox above the slider to input a custom value (it is editable).
